src/microbe_stage/editor/MicrobeEditorPatchMap.tscn

```diff
@@ -115,7 +115,6 @@ margin_left = 10.0
 margin_top = 10.0
 margin_right = 896.0
 margin_bottom = 608.0
-ShowScrollbars = false
```

Comment:
What's this change about?

Reply:
Looks like Godot is automatically changing things again... I'll revert this